app/src/main/java/org/wikipedia/page/leadimages/PageHeaderView.java

```diff
@@ -138,6 +138,7 @@ private void init() {
         ButterKnife.bind(this);
         ViewCompat.setTransitionName(this, getContext().getString(R.string.transition_floating_queue));
         gradientView.setBackground(getPowerGradient(R.color.black38, Gravity.TOP));
+        gradientView.setBackground(getPowerGradient(R.color.black38, Gravity.BOTTOM));
```

Comment:
Not related to this, but I found that in dark mode, the text is almost unseeable. 
Should we change the text and the icon color to be just white? (or ask Robin for the suggestion)